src/components/HoverArrow.tsx

```diff
@@ -0,0 +1,36 @@
+import { token } from '../../styled-system/tokens/index.mjs'
+
+interface HoverArrowType {
+  hoverArrowVisibility: 'above' | 'below' | null
+  scrollTop: number
+  arrowBottom: number
+}
+
+/** Renders upward/downward arrow when hovering over a sorted context. */
+const HoverArrow = ({ hoverArrowVisibility, scrollTop, arrowBottom }: HoverArrowType) => {
+  return (
+    hoverArrowVisibility && (
+      <div
+        style={{
+          width: '0',
+          height: '0',
+          borderLeft: '10px solid transparent',
+          borderRight: '10px solid transparent',
+          position: 'absolute',
+          top: hoverArrowVisibility === 'above' ? scrollTop : undefined,
+          left: '50%',
+          zIndex: '99999',
```

Comment:
Could you please assign a zIndex from the Panda config? We're trying to keep all of the zIndex values in one place for consistency and to avoid accidental conflicts: https://github.com/cybersemics/em/blob/57bed066075268fc63c1694bc5a05c666f506274/panda.config.ts#L251-L275

Reply:
Sure, thanks for pointing out.